run-test.js

```diff
@@ -0,0 +1,72 @@
+// const exec =  require('child_process').exec;
+const { spawn, exec } = require('node:child_process');
+const {echo} = require('shelljs');
+const minimist = require("minimist");
+const chalk = ("chalk");
+
+const log = console.log;
+// const error = chalk.bold.red;
+// const debug = chalk.bold.green;
+// const info = chalk.rgb(215, 135, 95);
+
+echo("Running Script")
+exec("jest -u --colors --changedSince feature/patial-test") //this works fine
```

Comment:
first try to get base and target branch using minimist. if they are empty then use gh cli to find it(in case of local machine) otherwise use them to form this command(in case of ci).